lstchain/calib/camera/tests/test_pedestals.py

```diff
@@ -26,53 +27,32 @@ def test_pedestal_calculator():
     subarray.tel[0].camera.readout.reference_pulse_shape = np.ones((1, 2))
     subarray.tel[0].camera.readout.reference_pulse_sample_width = u.Quantity(1, u.ns)
 
-
+    config = Config({
+        "FixedWindowSum": {
+          "apply_integration_correction": False,
+        }
+    })
     ped_calculator = PedestalIntegrator(charge_product="FixedWindowSum",
+                                        config=config,
                                         sample_size=n_events,
                                         tel_id=tel_id,
                                         subarray=subarray)
     # create one event
-    data = EventAndMonDataContainer()
+    data = ArrayEventContainer()
     data.meta['origin'] = 'test'
 
     # fill the values necessary for the pedestal calculation
     data.mon.tel[tel_id].pixel_status.hardware_failing_pixels = np.zeros(
         (n_gain, n_pixels), dtype=bool
     )
     data.r1.tel[tel_id].waveform = np.full((2, n_pixels, 40), ped_level)
-    data.r1.tel[tel_id].trigger_time = 1000
-
+    data.trigger.time = Time(0, format='mjd', scale='tai')
     while ped_calculator.num_events_seen < n_events:
         if ped_calculator.calculate_pedestals(data):
             assert data.mon.tel[tel_id].pedestal
             assert np.mean(data.mon.tel[tel_id].pedestal.charge_median) == (
-                ped_calculator.extractor.window_width.tel[0] * ped_level
+                ped_calculator.extractor.window_width.tel[tel_id] * ped_level
             )
             assert np.mean(data.mon.tel[tel_id].pedestal.charge_std) == 0
 
 
-def test_calc_pedestals_from_traces():
```

Comment:
Why remove the test?

Reply:
this is a test of a code that is in ctapipe, not needed here

Reply:
ok

Reply:
the test is already there